csrc/dd_arith.c

```diff
@@ -181,6 +182,11 @@ ddouble expm1q(ddouble a)
     return subqd(sum, 1.0);
 }
 
+ddouble inline ldexpqi(ddouble a, int exp)
```

Comment:
You need to remove inline here ... C is very peculiar.
```suggestion
ddouble ldexpqi(ddouble a, int exp)
```

Reply:
`inline` removed